docs/architecture/microservices/net-core-net-framework-containers/net-framework-container-scenarios.md

```diff
@@ -35,8 +35,6 @@ The following list shows most of the technologies that are not available in .NET
 
```

Comment:
line 32 mentions 2017. Should probably be updated.

Reply:
Yep, should be "since"

Reply:
Updated to as of Feb-2020